views/backend/postback.php

```diff
@@ -116,7 +47,7 @@
                 <div class="switch">
                     <label>
                         Non
-                        <input type="checkbox" name="public" <?php echo ($post->posted == "1")?"checked" : "" ?>/>
+                        <input type="checkbox" name="public" <?php echo ($post->getPosted() == "1")?"checked" : "" ?>/>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)